hypha/apply/api/v1/filters.py

```diff
@@ -29,10 +31,23 @@ class SubmissionsFilter(filters.FilterSet):
         field_name='page', label='fund',
         queryset=Page.objects.type(FundType) | Page.objects.type(LabType)
     )
+    screening_statuses = filters.ModelMultipleChoiceFilter(
+        field_name='screening_statuses',
+        queryset=ScreeningStatus.objects.all(),
+    )
+    reviewers = filters.ModelMultipleChoiceFilter(
+        field_name='reviewers',
+        queryset=get_user_model().objects.all(),
+    )
+    lead = filters.ModelMultipleChoiceFilter(
+        field_name='lead',
+        queryset=get_user_model().objects.all(),
+    )
```

Comment:
Can we use the queryset which are already being used for [Django table filters](https://github.com/OpenTechFund/hypha/blob/f25b41b58eb3f54cee20924af89be20be85020c2/hypha/apply/funds/tables.py#L262)?

https://github.com/OpenTechFund/hypha/blob/f25b41b58eb3f54cee20924af89be20be85020c2/hypha/apply/funds/tables.py#L202

https://github.com/OpenTechFund/hypha/blob/f25b41b58eb3f54cee20924af89be20be85020c2/hypha/apply/funds/tables.py#L197

https://github.com/OpenTechFund/hypha/blob/f25b41b58eb3f54cee20924af89be20be85020c2/hypha/apply/funds/tables.py#L193

As not all users are reviewers and leads.

Reply:
@sks444 , It is a filterset class, here we don't have access `request` object, so we can't use these functions.